extensions/wikia/CategoryPage3/CategoryPage3Hooks.class.php

```diff
@@ -123,13 +123,51 @@ public static function onGetPreferences( $user, &$defaultPreferences ) {
 		return true;
 	}
 
-	static public function onUserGetDefaultOptions( &$defaultOptions ) {
+	public static function onLinkerMakeExternalLink( string $url, string $text, string &$link, array $attribs ): bool {
+		global $wgCityId, $wgContLang, $wgFandomBaseDomain, $wgWikiaBaseDomain;
+
+		$host = parse_url( $url, PHP_URL_HOST );
+		$host = wfNormalizeHost( $host );
+
+		// External-external links
+		if ( !endsWith( $host, $wgFandomBaseDomain ) && !endsWith( $host, $wgWikiaBaseDomain ) ) {
+			return true;
+		}
+
+		// Not this wiki
+		if ( WikiFactory::UrlToID( $host ) !== $wgCityId ) {
+			return true;
+		}
+
+		$path = parse_url( $url, PHP_URL_PATH );
+		$categoryNsText = $wgContLang->getNsText( NS_CATEGORY );
+
+		// Not a category
+		if ( strpos( $path, $categoryNsText ) === false ) {
```

Comment:
Maybe include the colon here just in case "Category" as a string appears in another page title?